contentcuration/contentcuration/frontend/channelEdit/views/ImportFromChannels/ImportContentProgressModal.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+
+  <VDialog
+    :value="true"
+    persistent
+    width="50%"
+    @keydown.esc="handleCancel"
+  >
+    <VCard class="pa-2">
+      <VCardTitle>
+        <h2> {{ cardTitle }}</h2>
+      </VCardTitle>
+      <VCardText>
+        <p>{{ cardText }}</p>
+
+        <VLayout v-if="!isCanceling" row align-center>
+          <VProgressLinear color="secondary" :value="progress" />
+          <span class="ml-2">
+            {{ $formatNumber(progress / 100, { style: 'percent' }) }}
```

Comment:
When I tested, this was `NaN` as a result of whatever was going awry on the backend (maybe because I didn't have celery running or something)?  

Either way, it seems like it might be worthwhile to have a little bit of error handling for when the backend acts up.  Maybe in the next iteration?